ui/src/components/payment/payment-dialog/PaymentDialog.tsx

```diff
@@ -253,14 +285,20 @@ export default class PaymentDialog extends Component<PaymentProps, PaymentState>
         );
     };
 
+    private calculatePrice = () => {
+        if (this.state.pricePerMonth == null) {
```

Comment:
We have `ObjectUtils.checkNotNull(...)` method.

Reply:
++